src/Microsoft.Diagnostics.Runtime/src/Builders/RuntimeBuilder.cs

```diff
@@ -146,13 +146,13 @@ public void Dispose()
                     {
                         // As long as we got at least one heap we'll count that as success
                         result = true;
-                        ProcessHeap(segBuilder, clrHeap, in heap, allocContexts, segs, finalizerRoots, finalizerObjects);
+                        ProcessHeap(segBuilder, clrHeap, heap, allocContexts, segs, finalizerRoots, finalizerObjects);
```

Comment:
ClrHeap is a relatively large struct, and passing it by read-only reference was intentional.  Before I review the rest of the changes, why are you proposing changing these to by-value?  My cutoff here is roughly "if a struct is larger than 3 pointers, consider passing a readonly by-ref instead of by-value".

Reply:
Ah, it's still passed by reference. The `in` modifier is unnecessary at the call site. We don't usually use that...